pyaedt/edb_core/siwave.py

```diff
@@ -112,13 +124,15 @@ def do_pin_grouping(self,value):
 
 
 class CircuitPort(Source):
+    """Manages circuit ports."""
```

Comment:
```suggestion
    """Manages a circuit port."""
```
To be consistent with the other boundary classes.